include/cucumber-cpp/internal/CukeDll.hpp

```diff
@@ -0,0 +1,14 @@
+#ifndef CUKE_CUKEDLL_HPP_
```

Comment:
Replacing completely with the generated done is okay with me. Please make that `target_include_directories` so that transitive build requirements work properly.